experimental/Schemes/CoveredScheme.jl

```diff
@@ -390,8 +390,8 @@ _compose_along_path(X::CoveredScheme, p::Vector{Int}) = _compose_along_path(X, [
    } <: AbsCoveredSchemeMorphism{
                                  DomainType,
                                  CodomainType,
-                                 CoveredSchemeMorphism,
-                                 BaseMorphismType
+                                 BaseMorphismType,
+                                 CoveredSchemeMorphism
```

Comment:
Are you repairing a previous inconsistency or is there some important reason behind the change of order of arguments.

Reply:
No. Just repair. It was wrong before.